### DISCUSSION TITLE
What is the "Pending" column in the version table of PR descriptions?

### DISCUSSION THREAD

**Discussion by rcdailey** [renovatebot/renovate#30171] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Below is a markdown snippet from one of my Renovate PRs to upgrade a docker image tag:

This PR contains the following updates:

| Package | Update | Change | Pending |
|---|---|---|---|
| ollama/ollama | minor | `0.1.48` -> `0.2.1` | `0.2.5` (+3) |

I have never seen the "Pending" column before. What does this represent? And what is the `+3` portion?

I googled, checked the docs, and searched the discussion forums here but didn't find anything.

Thanks in advance.

**Comment by rarkins** [comment, marked as answer]
It happens when you use minimumReleaseAge and one or more haven't met the threshold yet

**Reply by rcdailey** [reply]
Thank you. What is the +3?

**Reply by RahulGautamSingh** [reply]
It is the no. of pending versions. In this case 0.2.2, 0.2.3, 0.2.4 are the pending versions
